app/src/main/java/com/github/geohunt/app/utility/TaskUtils.kt

```diff
@@ -13,15 +13,24 @@ import kotlin.coroutines.cancellation.CancellationException
  * @param activity the activity in which the return CompletableFuture will be executed
  */
 fun <TResult> Task<TResult>.toCompletableFuture(activity: Activity) : CompletableFuture<TResult>
+{
+    return this.toCompletableFuture()
+}

```

Comment:
The activity here is sued to ensures that the listener get deleted when the activity get destroyed and should probably not be removed

Reply:
Done!